### PR TITLE
Replace old name with new name

### PR DESCRIPTION
**Pull request by fjgarlin**
Some of this changes are require after the change of name, otherwise some assets are not found.

Probably use this to update references from Drupal 8 to Drupal 9 too.

Tasks done:

- [x] Replace `drupal8ci` for `drupal9ci` where appropriate.
- [x] Replace `Drupal 8` for `Drupal` or `Drupal 9` where appropriate.
- [x] Adapted `Github actions` integration for Drupal 9. See [here](https://github.com/fjgarlin/d9-lagoon/actions/runs/734387025).
- [x] Removed the dependency on the image with the database to build the actions. Now it tasks `mariadb` and installs the database before running unit tests.

### CODE REVIEW

**Comment by juampynr** [comment]
Looks good to me so far, @fjgarlin.

**Comment by fjgarlin** [comment]
Thanks, I think I've got a more simplified version of github actions working now too.
Happy for you to review.

File: README.md [hunk 1]
```diff
@@ -65,14 +65,12 @@ The [CircleCI configuration file](dist/circleci/.circleci/config.yml) uses a
...
-does not fit your project's architecture, then have a look at its
-[Dockerfile](https://github.com/Lullabot/drupal8ci/blob/master/circleci/.circleci/images/primary/Dockerfile)
-and consider [creating your own image](https://circleci.com/docs/2.0/custom-images/)
+does not fit your project's architecture then consider [creating your own image](https://circleci.com/docs/2.0/custom-images/)
 based out of it.
    
 #### Setting up the update path
 
-The Behat job requires a running Drupal 8 site. The repository contains the code, but for running
+The Behat job requires a running Drupal 9 site. The repository contains the code, but for running
```
Comment (juampynr) [inline README.md:73]: @fjgarlin I know that you like Crypress.io (https://events.drupal.org/europe2020/sessions/add-end-end-tests-your-drupal-site-one-hour-cypressio). I love it. I would love to replace the Behat tests by Cypress.io ones. Will create an issue for it.

Comment (juampynr) [reply README.md:73]: Here it is https://github.com/Lullabot/drupal9ci/issues/67

File: dist/github-actions/.github/workflows/ci.yml [hunk 2]
```diff
@@ -8,9 +8,12 @@ jobs:
...
     services:
       mariadb:
-        image: quay.io/juampynr/drupal8-github-actions:latest
+        image: mariadb:latest
+        env:
+          MYSQL_ROOT_PASSWORD: root
         ports:
-          - 33306:3306
+          - 3306:3306
+        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
```
Comment (juampynr) [inline dist/github-actions/.github/workflows/ci.yml:16]: 💯

**Comment by juampynr** [comment]
Looks great! Merging.

File: README.md (hunk 1 above)
Comment (fjgarlin) [reply README.md:73]: Great!

**Pull request merged**